app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -788,6 +806,41 @@ public void execute(Realm bgRealm) {
                                             answer = finalSetAlarm;
                                             isWebSearch = false;
                                         }
+                                        if(finalReminder != null && finalReminder.equals(getString(R.string.reminder_description))){
+                                            Log.d(TAG , "reminder Counter  " + reminderQuery);
+                                            answer = finalReminder;
+                                            reminderQuery =0;
+                                            isWebSearch = false;
+
+
+                                        }
+                                        else {
+                                            switch(reminderQuery){
+                                                case 0:{
```

Comment:
Brackets are not needed for cases inside switch cases, a case starts from `:` and ends in `break;` so it would have been better if you have removed them.